src/applications/facility-locator/containers/FacilityLocatorApp.jsx

```diff
@@ -1,29 +1,44 @@
 /* eslint-disable prettier/prettier */
 import { connect } from 'react-redux';
-import { Link, browserHistory } from 'react-router';
+import { Link } from 'react-router';
 import React from 'react';
 import DowntimeNotification, {
   externalServices,
 } from '../../../platform/monitoring/DowntimeNotification';
 import Breadcrumbs from '@department-of-veterans-affairs/formation/Breadcrumbs';
 import { ccLocatorEnabled } from '../config';
-
-/**
- * Preserves the search form in the UI & address bar when
- * navigating back to the search/map page.
- *
- * @param {Object} e The click event
- */
-const goBackHistory = (e) => {
-  e.preventDefault();
-  browserHistory.goBack();
-};
+import appendQuery from 'append-query';
 
 class FacilityLocatorApp extends React.Component {
+  buildSearchString() {
```

Comment:
This is great, can you please add a TODO comment to set this into a shared helper, so that it can be reused in VAMap which currently has a comment to try out using appendQuery?

Reply:
@cehsu And added. I'll let the tests run again and merge after all lights green. Thank you again for pair coding and review.